Update readme.md

benodonohue opened the pull request:
I have made a few changes as i found a few minor issues when attempting the installation instructions. Now using HTTPS and have added `npm install` to the standalone sprout cli.

benodonohue committed:
Update readme.md

I have made a few changes as i found a few minor issues when attempting the installation instructions.

jescalan commented on readme.md:
```diff
@@ -27,3 +27,52 @@ This is the default template for use within [spike](https://github.com/static-de
+- `cd <projectname>`
+- `npm install`
+- `spike watch`
+
```
So it looks like rather than changing the existing readme, you copy/pasted a full copy of the readme's content below the existing content. Could you try to merge them so there's just one copy of the readme text?

benodonohue replied:
Ok I will try to give it go mate.

jescalan replied:
Thank you!

benodonohue committed:
Update readme.md

Is this right? I rewrote the whole thing as i noticed that some of the terminal commands were out of order..

jescalan commented on readme.md:
```diff
@@ -8,22 +8,39 @@ An alternate template for [spike](https://github.com/static-dev/spike) that uses
-## Options
+- `cd <projectname>`
+- `npm install`
+- `spike watch`
```
These sections here are still doubled up, although this is much better than the last round!

benodonohue committed:
Update readme.md

jescalan commented on readme.md:
```diff
@@ -8,22 +8,36 @@ An alternate template for [spike](https://github.com/static-dev/spike) that uses
+	? What is the name of your project? 
+	? Describe your project 
+	? What is your github username? 
+	? Would you like a production config file? (Y/n)
```
Almost there, this last section here is repeated!

benodonohue replied:
Haha! I'm learning slowly.. will try again.

benodonohue replied:
I just realised that it's meant to be repeated because one is for the spike install of roots template and one for the sprout install of roots..

benodonohue replied:
I hope that makes sense. Should i just start from scratch with a fresh PR?
